.github/workflows/build.yml

```diff
@@ -285,12 +282,12 @@ jobs:
           echo "Restore"
           dotnet restore
           echo "Clean"
-          dotnet clean -c "Release" -o obj/Azure/publish
```

Comment:
Since we aren't using docker in this job, I think we should switch this one back.

Reply:
Updated path to `obj/build-output/publish`.